src/components/RatingStars/RatingStars.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './RatingStars.component.scss';
+class RatingStarsComponent extends React.Component {
+	constructor(props) {
+		super(props);
```

Comment:
Unless performing extra steps inside of the `constructor`, it can be left out.

Reply:
Removed constructor, and updated it as functional component